### DISCUSSION TITLE
Is it possible to change icons of a cover entity?

### DISCUSSION THREAD

**Discussion by SanderM2** [Mariusthvdb/custom-ui#139] [General]
I'm trying to figure out if it's possible to change the default icons home assistant uses for covers. I want to change the default icons for UP, STOP and DOWN to mdi:arrow-up-bold-circle-outline, mdi:stop-circle-outline and mdi:arrow-down-bold-circle-outline respectively.

I was unable to find a way to do this and now someone said I can probably use "custom-ui" for this but I have no experience with this addon.

Can you please tell me if this would be possible and maybe how?

**Comment by Mariusthvdb** [comment]
Sure, check the examples.
